lib/src/lints/function_lines_of_code/models/function_lines_of_code_parameters.dart

```diff
@@ -1,26 +1,27 @@
+import 'package:solid_lints/src/common/parameters/excluded_identifiers_list_parameter.dart';
+
 /// A data model class that represents the "function lines of code" input
 /// parameters.
 class FunctionLinesOfCodeParameters {
   /// Maximum allowed number of lines of code (LoC) per function,
   /// exceeding this limit triggers a warning.
   final int maxLines;
 
-  /// Function names to be excluded from the rule check
-  final List<String> excludeNames;
+  /// A list of methods that should be excluded from the lint.
+  final ExcludedIdentifiersListParameter exclude;
 
   static const _defaultMaxLines = 200;
 
   /// Constructor for [FunctionLinesOfCodeParameters] model
   const FunctionLinesOfCodeParameters({
     required this.maxLines,
-    required this.excludeNames,
+    required this.exclude,
   });
 
   /// Method for creating from json data
   factory FunctionLinesOfCodeParameters.fromJson(Map<String, Object?> json) =>
       FunctionLinesOfCodeParameters(
         maxLines: json['max_lines'] as int? ?? _defaultMaxLines,
-        excludeNames:
-            List<String>.from(json['excludeNames'] as Iterable? ?? []),
+        exclude: ExcludedIdentifiersListParameter.defaultFromJson(json),
```

Comment:
This is a breaking change. Changing it into the consistent naming probably does make sense, but we should bump the minor version and write it out in the CHANGELOG.